### ISSUE TITLE
Use G Maps for showing house address

### ISSUE THREAD

**Issue by BerkhanBerkdemir** [BerkhanBerkdemir/magara#16]

**Comment by noloman** [comment]
Hi @BerkhanBerkdemir can you please give more info about this issue?

**Comment by BerkhanBerkdemir** [comment]
Of course, @noloman, but before that, I want you to show another issue which is #44. I wrote some details there about mapping. Actually, I could close that issue :smile:
